packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -419,10 +419,7 @@ module.exports = ({ model, strapi }) => {
   function count(params) {
     const filters = convertRestQueryParams(params);
 
-    return buildQuery({
-      model,
-      filters: { where: filters.where },
-    }).count();
+    return buildQuery({ model, filters }).count();
```

Comment:
you need to remove the limit / start / sort

Reply:
we do this in bookshelf `const filters = convertRestQueryParams(_.omit(params, ['_sort', '_limit', '_start']));`

Reply:
Make sense, I was about to ping you on why we were only passing the where to the count. Will add that

Reply:
Fixed in 266bc51